source/clear-linux/guides/deploy-at-scale.rst

```diff
@@ -0,0 +1,342 @@
+.. _deploy-at-scale.rst:
+
+Deploying |CL| at Scale
+#######################
+
+Once you are comfrotable with `Clear Linux concepts`_, your next step
+as a system administraotr may be needing to understand how to deploy 
```

Comment:
administraotr mispelled.  Also, this sentence seems passive in how it is written